scripts/cnv_wdl/somatic/README.md

```diff
@@ -48,6 +48,7 @@ The reference used must be the same between PoN and case samples.
 - ``CNVSomaticPairWorkflow.targets`` -- (optional) Target file (NOT in bed format) that was used to describe the baits in capture (exome) samples.  Please run ``ConvertBedToTargetFile`` to convert a BED file to a target file.  If provided, then WES workflow will be run; otherwise, WGS workflow will be run.
 - ``CNVSomaticPairWorkflow.tumor_bam_idx`` -- File path or storage location (depending on backend) of the tumor BAM file index.
 - ``CNVSomaticPairWorkflow.tumor_bam`` -- File path or storage location (depending on backend) of the tumor BAM index.
+- ``CNVSomaticPairWorkflow.is_run_oncotator`` -- (optional default: false) Whether oncotator should be run on the called segment file.
```

Comment:
All Done, including conversation about having it only after total copy ratio (for now), since this is what is typically delivered to many users.  This should be temporary.

I am adding it to run in travis for somatic only.

Done